README.md

```diff
@@ -126,6 +126,69 @@ $ make CONF=linux-x86_64-normal-server-release THIRD_PARTY_HEAP=$PWD/../mmtk-ope
 
 The output jdk is then found at `./build/linux-x86_64-normal-server-release/images/jdk`.
 
+### Profile-Guided Optimized Build
+
+In order to get the best performance, we recommend using a profile-guided
+optimized (PGO) build. Rust supports [PGO
+builds](https://doc.rust-lang.org/rustc/profile-guided-optimization.html) by
+directly hooking into the LLVM profiling infrastructure. In order to have the
+correct LLVM tools version, you should install the relevant `llvm-tools-preview`
+component using `rustup`:
+
+```console
+$ rustup component add llvm-tools-preview
+```
+
+We use `fop` as it is a relatively small benchmark but also exercises the GC.
```

Comment:
This is specific to DaCapo, and specific to our approach. Ideally, you should run profiling with the exact workload you want to benchmark. Using `fop` for DaCapo is just our workaround to avoid running profiling for the whole DaCapo suite. It is okay to use `fop` as an example for the rest of the text. But probably make it clear here that for the best result, you should run the workload that you try to benchmark and use the exact MMTk plan you try to benchmark with.

Reply:
But for a general JVM build, we still have to profile against something. `fop` was just chosen as a general representative benchmark and `GenImmix` is one of our best-performing collectors. I would argue that if you tune too much to a particular benchmark, that's essentially giving MMTk an unfair edge as G1 and other GCs don't do PGO (or at least to the best of my knowledge).

Reply:
You can keep most of the text here about `fop` and `GenImmix`. You just need to make it clear that this is just an example. As a user, you should profile with the plan you are using, and with the workload that you care about.